node_modules/col-assets/tests/test-assets.js

```diff
@@ -857,18 +857,18 @@ describe('Assets', function() {
       TestsUtil.getAssetLibraryClient(null, null, null, function(client, course, user) {
 
         // Create a visible UC Berkeley file asset
-        AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Visible', getFileStream('logo-ucberkeley.png'), null, function(visisbleBerkelyFile) {
+        AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Visible', getFileStream('logo-ucberkeley.png'), null, function(visisbleBerkeleyFile) {
           // Create a visible UC Berkeley link asset
-          AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Visible', 'http://www.berkeley.edu/', null, function(visisbleBerkelyLink) {
+          AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Visible', 'http://www.berkeley.edu/', null, function(visisbleBerkeleyLink) {
             // Create a visible UC Davis asset
             AssetsTestUtil.assertCreateLink(client, course, 'UC Davis Link Visible', 'http://www.davis.edu/', null, function(visisbleDavisLink) {
               // Create a hidden UC Berkeley file asset
-              AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Hidden', getFileStream('logo-ucberkeley.png'), {'visible': false}, function(hiddenBerkelyFile) {
+              AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Hidden', getFileStream('logo-ucberkeley.png'), {'visible': false}, function(hiddenBerkeleyFile) {
                 // Create a hidden UC Berkeley link asset
-                AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Hidden', 'http://www.berkeley.edu/', {'visible': false}, function(hiddenBerkelyLink) {
+                AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Hidden', 'http://www.berkeley.edu/', {'visible': false}, function(hiddenBerkeleyLink) {
 
                   // Verify that searching through the assets doesn't return hidden assets
-                  verifySearch(client, course, {'keywords': 'berkeley'}, [visisbleBerkelyLink, visisbleBerkelyFile], function() {
+                  verifySearch(client, course, {'keywords': 'berkeley'}, [visisbleBerkeleyLink, visisbleBerkeleyFile], function() {
```

Comment:
@nicolaasmatthijs Somehow one typo kept me from seeing another (`visisble`). Sorry to have missed it on last review!